ta/aes_perf/include/user_ta_header_defines.h

```diff
@@ -32,7 +32,10 @@
 
 #define TA_UUID TA_AES_PERF_UUID
 
-#define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR)
+#define TA_FLAGS                (TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR | \
+				TA_FLAG_SECURE_DATA_PATH | \
+				TA_FLAG_CACHE_MAINTENANCE)
```

Comment:
I felt it was more clear to have these flags always set. If SDP is not supported, the TA implementation will reflect it. We could change this if you feel it will be easier to maintain.